uber/models.py

```diff
@@ -1324,6 +1348,22 @@ def past_years_json(self):
         return json.loads(self.past_years or '[]')
 
 
+class WatchList(MagModel):
```

Comment:
I don't think so, after re-reading everything it looks good to me, so after fixing the ``relationship`` definition I think it's good to merge.